buildenv/jenkins/jobs/infrastructure/wrapper_template.groovy

```diff
@@ -59,53 +58,53 @@ pipelineJob(job_name){
             lightweight(true)
         }
     }
-    if (build_discarder_logs || build_discarder_artifacts){
-        logRotator{
-            if (build_discarder_logs){
+    if (build_discarder_logs || build_discarder_artifacts) {
+        logRotator {
+            if (build_discarder_logs) {
                 numToKeep(build_discarder_logs)
             }
-            if (build_discarder_artifacts){
+            if (build_discarder_artifacts) {
                 artifactNumToKeep(build_discarder_artifacts)
             }
         }
     }
-    if (github_project){
+    if (github_project) {
         properties {
             githubProjectUrl(github_project)
         }
     }
-    if (parameters){
+    if (parameters) {
         parameters {
-            parameters.boolean_parameters.each{ boolean_parameter ->
-                booleanParam{
+            parameters.boolean_parameters.each { boolean_parameter ->
+                booleanParam {
                     name(boolean_parameter.key)
                     defaultValue(boolean_parameter.value)
                     description(parameter_descriptions.get(boolean_parameter.key))
                 }
             }
             parameters.string_parameters.each { string_parameter ->
-                stringParam{
+                stringParam {
                     name(string_parameter.key)
                     defaultValue(string_parameter.value)
                     description(parameter_descriptions.get(string_parameter.key))
                     trim(true)
                 }
             }
-            parameters.choice_parameters.each{ choice_parameter ->
-                choiceParam{
+            parameters.choice_parameters.each { choice_parameter ->
+                choiceParam {
                     name(choice_parameter.key)
                     choices(choice_parameter.value)
                     description(parameter_descriptions.get(choice_parameter.key))
                 }
             }
         }
     }
-    if (triggers){
-        triggers{
-            if (triggers.cron){
+    if (triggers) {
+        triggers {
+            if (triggers.cron) {
                 cron(triggers.cron)
             }
-            if (triggers.pull_request_builder){
+            if (triggers.pull_request_builder) {
                 githubPullRequest {
                     admins(triggers.pull_request_builder.admin_list)
                     blackListCommitAuthor(triggers.pull_request_builder.blacklist)
```

Comment:
Changing this file (DSL Template) will likely trigger a script approval which we'll need an Eclipse bugzilla for.

Reply:
Is it sufficiently difficult to get that approval that we should not change this?

Reply:
~No we can raise the request, just need to get it pending so it can be approved. I believe it should trigger if we do a PR build.~
Ah, scratch that, this is the wrapper template, we don't actually use this file.

Reply:
Should we just remove this file if it's not used?

Reply:
Well, it could be used. May need to be updated, but we don't use it on a regular basis.

Reply:
@AdamBrousseau Is there a document somewhere that explains which files require Eclipse approval to change? If not, could you open a PR to add that information, please?

Reply:
https://github.com/eclipse/openj9/pull/10050 was opened.